### ISSUE TITLE
Incorrect double click behavior in Custom Templates window

### ISSUE THREAD

**Issue by akolyakov42** [epam/ketcher#1364]
**Steps to Reproduce**
1. Launch Ketcher
2. Open 'Custom Templates' window (button in bottom toolbar)
3. Select unselected template with single click
4. Double click on selected template.

**Expected behavior**
Template window is closed and next click on desirable location on canvas will add template to this location.

**Actual behavior**
Template window is closed with first click on selected template and second click adds template on canvas in position under the cursor.

**Screencast**
https://user-images.githubusercontent.com/97884482/160374452-d92a83c6-062f-4f72-a96b-04a7d54ce448.mov

**User agent**
Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7)
AppleWebKit/537.36 (KHTML, like Gecko)
Chrome/98.0.4758.109
Safari/537.36

**Ketcher version**
v2.4.2-136-g178fc6ab

**Additional context**
Double click on selected template must work as OK button in down right corner of Custom Templates window

**Comment by fedorseliankin** [comment]
Agreed with @Alena-Guznischeva the following changes. Single click on the selected template does not close the template selection window. Double click on unselected or selected template works as OK button in the down right corner. This behavior is repeated in periodic table window only in "single" mode

**Comment by Svetlana-Asinovskaia** [comment]
Verified in Release/2.5.0
